Update 11-16-21-alueker-project-update-and-schedule

alueker opened the pull request:
Changed author name to correct name, assignment should appear on website after the edit

alueker committed:
Update 11-16-21-alueker-project-update-and-schedule

Changed author name to correct name, assignment should appear on website after the edit

MishaCienfuegos commented on _posts/alueker/11-16-21-alueker-project-update-and-schedule:
```diff
@@ -1,6 +1,6 @@
 ---
 layout: post
-author: alueker
+author: Andrew Lueker
```
For your post to show up under your name on the People page, you will want the author to match your alias in the `config.yml` [file](https://github.com/inf380p/inf380p.github.io/blob/production/_config.yml#L107). It looks to me like they match up fine before this change. Does that seem true?

alueker replied:
I believe I changed my alias after the first four assignments were uploaded. Double check it but I changed my name Andrew Lueker and I didn't see the project proposal or project update get uploaded to the site.

MishaCienfuegos replied:
On line 107 of the [config.yml](https://github.com/inf380p/inf380p.github.io/blob/production/_config.yml#L107) it looks like your alias is still `alueker`. Did you see my note above about the file name, especially the date and file extension? That will also mess with how and where your post shows up on the website.

If you want to test the change as is,  we can merge and see if that resolves the problem of your November 16th update not showing up. But I still recommend those changes to the file name!

alueker replied:
I see what you mean, I think you're right, I believe the issue was not adding the .md at the end of the file name

MishaCienfuegos replied:
Great, that should help format your post as markdown!

You may also want to change the date format in the file name to `YYYY-MM-DD-` format. That specific syntax is important to the code that builds the website; without it, your post won't show up in the expected places.

alueker replied:
Okay, date changed, everything should be good to go now!

MishaCienfuegos reviewed (changes requested):
It looks to me like there are a few changes that should be made to the file title so that your post displays properly on the class website. Check out the structure of the titles of [your other posts](https://github.com/inf380p/inf380p.github.io/tree/production/_posts/alueker), particularly the date and file extension.

alueker committed:
Update and rename 11-16-21-alueker-project-update-and-schedule to 11-16-21-alueker-project-update-and-schedule.md

Changed the name back, I think the issue might be that I didn't add the .md at the end

alueker reviewed (commented):
This is more concise

alueker committed:
Rename 11-16-21-alueker-project-update-and-schedule.md to 2021-11-16-alueker-project-update-and-schedule.md

Markdown format and correct date format

eah13 commented on _posts/alueker/2021-11-16-alueker-project-update-and-schedule.md:
```diff
@@ -9,10 +9,10 @@ In my previous post I made a bold claim about the schedule of my game, a fully i
-- Create a timer that gradually degrades (gets faster as game progresses)
-- Create a score to count how many correct commands were performed in a row 
-- Create a score max that leads to a win screen
+1 Align button commands (i.e. space bar for Bop, up key for Spin, down for Pull, etc.) with in-game commands
+2 Create a timer that gradually degrades (gets faster as game progresses)
+3 Create a score to count how many correct commands were performed in a row 
+4 Create a score max that leads to a win screen
```
For an ordered list in markdown, you'll want a `.` after the number:

1 this is just the numeral

1. This has a trailing period.

alueker replied:
Okay, fixed the periods, should be good to launch now.

eah13 commented:
@alueker nice work sticking with these changes. One final suggestion is to add periods so you get a nice list on your post. Other than that I think this is ready to go.

alueker committed:
Update 2021-11-16-alueker-project-update-and-schedule.md

Added periods after numbers, should be good to go now.

MishaCienfuegos reviewed (approved):
Thanks for sticking with us through these edits. I think this looks great!
